### ISSUE TITLE
tag closed too late when followed by child with block expansion with include

### ISSUE THREAD

**Issue by rashfael** [rashfael/eslint-plugin-vue-pug#14]
`eslint-plugin-vue-pug` version 0.5.3

Error at `template(v-else)`:

```pug
div
	template(v-if='true')
		span: include ./assets/cross.svg
		span
	template(v-else)
		span
```

```
ESLint: 'v-else' directives require being preceded by the element which has a 'v-if' or 'v-else-if' directive.(vue/valid-v-else)
```

No error:

```pug
div
	template(v-if='true')
		span: include ./assets/cross.svg
	template(v-else)
		span
```

No error:

```pug
div
	template(v-if='true')
		span
			include ./assets/cross.svg
		span
	template(v-else)
		span
```

No error:

```pug
div
	template(v-if='true')
		span
	template(v-else)
		span
```

_Originally posted by @Grawl in https://github.com/rashfael/eslint-plugin-vue-pug/issues/13#issuecomment-1172137388_

**Comment by rashfael** [comment]
fixed in 0.5.4

**Comment by Grawl** [comment]
I can confirm that
